CharacterMap/CharacterMap/Views/FontMapView.xaml.cs

```diff
@@ -213,6 +213,22 @@ private void PreviewGrid_SizeChanged(object sender, SizeChangedEventArgs e)
                 item.Width = newSize;
             }
         }
+
+        internal void SearchBoxUnicode_SuggestionChosen(AutoSuggestBox sender, AutoSuggestBoxSuggestionChosenEventArgs args)
+        {
+            if (args.SelectedItem is IGlyphData data)
+            {
+                SelectCharacter(ViewModel.Chars.First(c => c.UnicodeIndex == data.UnicodeIndex));
+            }
+        }
+
+        internal void SearchBoxUnicode_GotFocus(object sender, RoutedEventArgs e)
+        {
+            if (ViewModel.SearchResults != null && ViewModel.SearchResults.Count > 0)
+                SearchBoxUnicode.IsSuggestionListOpen = true;
+            else
+                ViewModel.DebounceSearch(ViewModel.SearchQuery);
```

Comment:
May be we should start the search only when there are at least 2 chars in the search box?

Reply:
I did consider it, but in the end if you want to search  individual letter variations or punctuation marks you really want single character search